src/components/ec-letter-icon/ec-letter-icon.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div
+    data-test="ec-letter-icon"
+    class="ec-letter-icon"
+    :class="{ 'ec-letter-icon--clickable': isClickable }"
+  >
+    <svg
```

Comment:
why do we need svg here?

Reply:
SInce it's an icon I think it's better to have it as `svg` as we do with other icons (i.e. [ec-icon](https://github.com/Ebury/chameleon/blob/master/src/components/ec-icon/ec-icon.vue#L8) or [ec-timer](https://github.com/Ebury/chameleon/blob/master/src/components/ec-timer/ec-timer.vue#L6))

Reply:
We had to display complex things as circle or picture there, but here it's just one Letter is centred in the round container.